src/github.com/matrix-org/dendrite/roomserver/input/events.go

```diff
@@ -103,13 +107,84 @@ func processRoomEvent(db RoomEventDatabase, ow OutputRoomEventWriter, input api.
 		return err
 	}
 
-	// TODO:
-	//  * Caculate the new current state for the room if the forward extremities have changed.
-	//  * Work out the delta between the new current state and the previous current state.
-	//  * Work out the visibility of the event.
-	//  * Write a message to the output logs containing:
-	//      - The event itself
-	//      - The visiblity of the event, i.e. who is allowed to see the event.
-	//      - The changes to the current state of the room.
```

Comment:
This isn't strictly related to this PR, but I think all of that TODO is out of date, so it might as well be deleted.